modules/vaos/app/services/vaos/appointment_service.rb

```diff
@@ -102,6 +94,20 @@ def pagination(pagination_params)
       }
     end
 
+    def partial_errors(response)
+      if response.status == 200 && response.body[:errors]
```

Comment:
Should this be `response.body[:errors]&.any?`?

Reply:
no, because VAMF actually drops null valued keys. However at the controller level when check to make see if there are errors, we're always passing an empty array if VAMF provided us with null. There we check for .any

but to your point, &.any? would work just as well.

Reply:
made this change anyway, because why not?